README.md

```diff
@@ -66,11 +66,18 @@ If you are not a member of ExpressVPN, you may set up your own Earthly satellite
 
 # Releasing
 
-This repository is a monorepo for two crates: `wolfssl-sys` and `wolfssl`. They should __always__ be released together. Since `wolfssl` depends on `wolfssl-sys`, they should be released in the below order:
+This repository is a monorepo for two crates: `wolfssl-sys` and `wolfssl`. It is required when releasing `wolfssl` that it depends on an version of `wolfssl-sys` which is up to date with `main`.
 
-1. Bump the crate version `wolfssl-sys`, update the version specified under `dependencies` in the `wolfssl` crate, and release it (Follow the section [Releasing a Single Crate](#releasing-a-single-crate))
-1. Bump the crate version `wolfssl` and release it (same procedure as above)
+If `wolfssl-sys` has unreleased changes:
```

Comment:
Might be useful to copy over the explanation of states from the PR description into here as well.